syntax = "proto3";

package freshCache;

service CacheService {
    rpc Get(CacheGetRequest) returns (CacheGetResponse);
    rpc Set(CacheSetRequest) returns (CacheSetResponse);
}

message CacheGetRequest {
    string key = 1;
}

message CacheGetResponse {
    string value = 1;
    bool success = 2;
}

message CacheSetRequest {
    string key = 1;
    string value = 2;
} 

message CacheSetResponse {
    bool success = 1;
}
